### PR TITLE
Caroline cm atkinson marriage abroad vietnam updates

### PR DESCRIPTION
**Pull request by CarolineCMAtkinson**
Trello: https://trello.com/c/aZXstlbY/2787-publishing-marriage-abroad-vietnam-changes

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Comment by CarolineCMAtkinson** [comment]
Updated 25 outcomes of marriage abroad Vietnam. 

https://trello.com/c/7PY9NX6B/3604-getting-married-abroad-vietnam-payment-and-ubnd-updates

**Comment by CarolineCMAtkinson** [comment]
** Converted this to draft as the department need to check this before we go live. **

File: app/flows/marriage_abroad_flow/outcomes/countries/vietnam/uk/partner_other/_same_sex.erb [hunk 1]
```diff
@@ -55,10 +55,9 @@ Receiving a notice of registration | <%= format_money calculator.consular_fee(:r
 Registering a marriage | <%= format_money calculator.consular_fee(:registering_marriage) %>
 Issuing a marriage certificate | <%= format_money calculator.consular_fee(:issuing_civil_partnership_or_marriage_certificate) %>
 
-You can pay fees for consular services in the local currency or pounds sterling - these are shown in the list of [consular fees for Vietnam](/government/publications/vietnam-consular-fees).
+Fees are shown in the list of [consular fees for Vietnam](/government/publications/vietnam-consular-fees). 
 
-<%=  render partial: 'how_to_pay',
-            locals: {calculator: calculator}%>
+It's quicker and easier to pay by card (Visa or Mastercard). You can also pay in cash if you pay in the local currency.
```
Comment (ChrisBAshton) [inline app/flows/marriage_abroad_flow/outcomes/countries/vietnam/uk/partner_other/_same_sex.erb:60]: I noticed that this is missing the link that is applied in `vietnam/_opposite_sex_civil_partnership.erb` (https://github.com/alphagov/smart-answers/blob/093bd82d488a2a9deea789dd02bcfcf85b9dce27/app/flows/marriage_abroad_flow/outcomes/countries/vietnam/_opposite_sex_civil_partnership.erb#L21). For consistency, do we want the link in all outcomes that use this text?

```suggestion
It's quicker and easier to pay by card (Visa or Mastercard). You can also pay in cash if you pay in the [local currency](/government/publications/vietnam-consular-fees).
```

**Review by ChrisBAshton — COMMENTED** [review]
Hello 👋  I've made a comment below, based on a quick check of the files. There may be other inconsistencies like that one, which is always likely to happen when there is so much repetition across files like this!

It might be possible for me to consolidate some of the repetition into a template, which we could reference in all relevant outcomes, and means you'd only need to edit that section in one place. For example, the Vietnam affirmation document section is long and looks identical (to me) between the different outcomes, so I could put all that into a `vietnam_affirmation_doc_section.erb`, if that would help ease the maintenance in future? It would be referenced something like:

```
<%=  render partial: 'vietnam_affirmation_doc_section' %>
```

...in the relevant part of the outcome. That would mean, however, that it would create more work if you needed that section to diverge between outcomes in future.

Is this something you'd like me to look into, or shall we stick with the self-contained outcomes approach?

File: app/flows/marriage_abroad_flow/outcomes/countries/vietnam/uk/partner_other/_same_sex.erb (hunk 1 above)
Comment (CarolineCMAtkinson) [reply app/flows/marriage_abroad_flow/outcomes/countries/vietnam/uk/partner_other/_same_sex.erb:60]: Hi Chris, thanks for checking and for spotting this! If we could add that link in, that would be great, thanks. Would you mind if we keep the self-contained outcomes approach for now, please? Many thanks

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**